packages/connect/test/http_client_test.ts

```diff
@@ -53,9 +53,9 @@ describe('HttpClient', () => {
                 assetDataA: assetData,
                 page: 3,
                 perPage: 50,
-                networkdId: 42,
+                chainId: 42,
             };
-            const urlWithQuery = `${url}?assetDataA=${assetData}&networkdId=42&page=3&perPage=50`;
+            const urlWithQuery = `${url}?assetDataA=${assetData}&chainId=42&page=3&perPage=50`;
```

Comment:
@dekz can you change the 0x API SRA endpoint to use `chainId` instead of `networkId`? We also need to update to SRA V3 spec with this change.